src/Facebook/FacebookBatchRequest.php

```diff
@@ -62,20 +62,21 @@ public function __construct(FacebookApp $app = null, array $requests = [], $acce
     }
 
     /**
-     * A a new request to the array.
+     * Adds a new request to the array.
      *
      * @param FacebookRequest|array $request
      * @param string|null           $name
+     * @param string[]              $options The batch options e.g. 'omit_response_on_success'
      *
      * @return FacebookBatchRequest
      *
      * @throws \InvalidArgumentException
      */
-    public function add($request, $name = null)
+    public function add($request, $name = null, array $options = [])
```

Comment:
It looks a bit tricky, but it works! Renaming `$name` to `$options` and document that if it's a string it will be the request name, otherwise it will be the array of options :)